src/main/java/sevensmurfs/rehub/repository/TherapyRepository.java

```diff
@@ -4,9 +4,11 @@
 import org.springframework.stereotype.Repository;
 import sevensmurfs.rehub.model.entity.Therapy;
 
+import java.util.List;
+import java.util.Optional;
 
 @Repository
 public interface TherapyRepository extends JpaRepository<Therapy, Long> {
 
-
+    Optional<List<Therapy>> findByPatientId(Long patiendId);

```

Comment:
Not sure will this work. I think it should be findByPatient(Patient patient);
Test it to see if it works.